pkgs/by-name/wa/warp-terminal/package.nix

```
@@ -2,11 +2,24 @@
 , stdenvNoCC
 , fetchurl
 , undmg
+, appimageTools
 }:
-stdenvNoCC.mkDerivation (finalAttrs: {
-  pname = "warp-terminal";
-  version = "0.2023.12.05.08.02.stable_00";
 
+let
+pname = "warp-terminal";
+version = "0.2023.12.05.08.02.stable_00";
+
+linux = appimageTools.wrapType2 {
+  inherit pname version;
+  src = fetchurl {
+    url = "https://app.warp.dev/download?package=appimage";
```

web.archive.org can help with that

```suggestion
    url = "https://web.archive.org/web/20240223005144/https://releases.warp.dev/stable/v0.2024.02.20.08.01.stable_01/Warp-x86_64.AppImage";
```

Oh? see, now we have a stable URL 😂 

```suggestion
    url = "https://releases.warp.dev/stable/v${version}/Warp-x86_64.AppImage";
```

---

Oh, thanks! How did you find this by the way?

---

@imadnyc If you `curl -v https://app.warp.dev/download?package=appimage` it shows you where it's redirecting